cpp/examples/abm_minimal.cpp

```diff
@@ -27,7 +27,7 @@ int main()
     mio::abm::GlobalInfectionParameters infection_params;
 
     // Set same infection parameter for all age groups. For example, the incubation period is 4 days.
-    infection_params.get<mio::abm::IncubationPeriod>() = 4.;
+    infection_params.get<mio::abm::IncubationPeriod>() = 4. * 24 * 3600;
```

Comment:
shouldn't doubles always be in days to be consistent with the ODE models?

Reply:
I had everything in seconds since this is the unit in `TimePoint` and `TimeSpan`. I see the point of changing it, though, so I changed it. This required me (or at least made it way easier) to introduce a `days(double)` method in `TimeSpan`. We should discuss if we want to have this and if yes we could potentially template it and also write it for the other functions.

Reply:
do you mean a function overload `TimeSpan days(double)` like `TimeSpan days(int)`? If so, I think this is a good overload, as long as `days(4.0)` and `days(4)` does the same thing. And I wouldn't template it. You can add hours(double) and minutes(double) if you like, but it's not necessary, since only days(double) is necessary to convert between ODE time and ABM time.

Ofcourse maybe others disagree about what unit they would expect a double time to be.

Reply:
Exactly. The functions are identical, except for a cast to int after the multiplication of the double with the number of seconds a day has.

As long as we are consistent with using day as our time unit for parameters, it is fine. Other units can always be converted.